pages/data.js

```diff
@@ -78,57 +78,30 @@ export default function DataForm() {
 
     return (
         <div>
-            <section className="flex flex-col items-center md:pt-5">
```

Comment:
Add padding or anything so that header theme btn & lang selector don't overlap on Mobile Screens

Reply:
add `pt-10` instead of `md:pt-5`